.eslintrc.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-env node*/
```

Comment:
small nit: missing space
```suggestion
/* eslint-env node */
```